Sajakhtar/cryptocanons
# Topicsapiseed

Added migration for Topic - new fields are symbol, cashtag, icon_url, coingecko_id
Updated seeds to fetch coins from Coinnmarketcap API, the icon_url and coingecko_id from Coingecko api.

To test:
Run rails db:migrate
Run rails db:seed - this will take 10+ mins
In localhost observe the homepage select now with all 200+ topics
